lib/views/menu-scene.js

```diff
@@ -69,6 +78,16 @@ class ListScene extends React.Component {
   onClosePress() {
     this.props.navigator.pop();
   }
+  onItemPress(id) {
+    const url = settings.links[id];
+    if (!url) {
+      return;
+    }
+
+    Linking.openURL(url).catch(err => {
+      console.info(`Could not open url ${url} `, err);
+    });
```

Comment:
Under what circumstance would this fail? I imagine only for malformed URLs. In which case, worth it?